lib/SPIRV/LLVMToSPIRVDbgTran.cpp

```diff
@@ -1612,19 +1612,21 @@ LLVMToSPIRVDbgTran::transDbgImportedEntry(const DIImportedEntity *IE) {
   // FIXME: 'OpenCL/bugged' version is kept because it's hard to remove it
   // It's W/A for missing 2nd index in OpenCL's implementation
   const SPIRVWord OffsetIdx =
-      isNonSemanticDebugInfo() ? OperandCount - NonSemantic::OperandCount : 0;
-  SPIRVWordVec Ops(OperandCount - OffsetIdx);
-  Ops[NameIdx] = BM->getString(IE->getName().str())->getId();
-  Ops[TagIdx] = SPIRV::DbgImportedEntityMap::map(Tag);
-  Ops[SourceIdx - OffsetIdx] = getSource(IE->getFile())->getId();
-  Ops[EntityIdx - OffsetIdx] = transDbgEntry(IE->getEntity())->getId();
-  Ops[LineIdx - OffsetIdx] = IE->getLine();
+      isNonSemanticDebugInfo()
+          ? OpenCL::OperandCount - NonSemantic::OperandCount
+          : 0;
```

Comment:
Does it make any sense to simplify this to ?
```suggestion
      isNonSemanticDebugInfo();
```

Reply:
Applied, thanks!